### DISCUSSION TITLE
Magic Link

### DISCUSSION THREAD

**Discussion by FemtDeveloper** [nextauthjs/next-auth#4905] [Help]
Hi, I have a custom Magic Link for email provider with  smtp.gmail.com, locally it works perfectly but when I deployed to vercel the link is not sended to the user, With Supabase for database
 My [...nextauth].js:

![image](https://user-images.githubusercontent.com/7281791/178386733-1201e930-bacb-43b5-8f28-76b32fb98c44.png)

**Comment by ThangHuuVu** [comment]
Hi, not sure what's the obstacle for you here, could you provide more details about the errors & include a minimal reproduction? 🙌
